### ISSUE TITLE
Immersive portals is incompatible with minecolonies

### ISSUE THREAD

**Issue by maximus1218** [qouteall/ImmersivePortalsMod#255]
While using this mod with minecolonies, no buildings can be placed. This makes minecolonies obsolete.

**Comment by LoganDark** [comment]
Many people have been trying minecolonies and reporting that it doesn't work. Did some new release come out recently or something?

**Comment by maximus1218** [comment]
No, I have been using minecolonies with immersive portals on 1.15.2. Immersive portals is using a coremod with mixins, which is doing something that blocks the placement of these necessary blocks in the minecolonies mod.

**Comment by LoganDark** [comment]
Of course, Forge. Transferring.
